src/views/CalendarView.vue

```diff
@@ -87,41 +87,47 @@ export default {
     }
   },
   computed: {
-    sortedTimes () {
+    sortedTimes() {
       return this.sortTimes()
     },
-    hasUserSchedule () {
+    hasUserSchedule() {
       return !_.isEmpty(this.availability)
     }
   },
-  created () {
+  created() {
     this.fetchData()
   },
   methods: {
-    fetchData () {
+    fetchData() {
       if (!this.user.hasSchedule) {
         CalendarService.initAvailability(this, this.user._id)
       }
 
-      var originalAvailabilityPromise = CalendarService.getAvailability(this, this.user._id)
+      var originalAvailabilityPromise = CalendarService.getAvailability(
+        this,
+        this.user._id
+      )
       var userTimezonePromise = CalendarService.getTimezone(this, this.user._id)
 
-      Promise
-        .all([originalAvailabilityPromise, userTimezonePromise])
-        .then(([originalAvailability, userTimezone]) => { 
+      Promise.all([originalAvailabilityPromise, userTimezonePromise]).then(
```

Comment:
I kinda prefer the previous formatting where `.all()` and `.then()` could each have their own line. What do you think?

Is this a prettier rule or something?

Reply:
Prettier doesn't seem to have any options for function chaining. It is quite opinionated (which is great, most of the time...). In this case, Prettier looks to be trying to minimize indentation. I like it.